docs/whl.md

```diff
@@ -32,9 +32,9 @@ This rule defines `@foo//:pkg` as a `py_library` target.
 | Name  | Description | Type | Mandatory | Default |
 | :-------------: | :-------------: | :-------------: | :-------------: | :-------------: |
 | name |  A unique name for this repository.   | <a href="https://bazel.build/docs/build-ref.html#name">Name</a> | required |  |
-| extras |  A subset of the "extras" available from this &lt;code&gt;.whl&lt;/code&gt; for which &lt;code&gt;requirements&lt;/code&gt; has the dependencies.   | List of strings | optional | [] |
+| extras |  A subset of the "extras" available from this <code>.whl</code> for which <code>requirements</code> has the dependencies.   | List of strings | optional | [] |
```

Comment:
IIUC this document is converted to HTML so these tags do something in the final document?

Reply:
Yeah. Viewing these docs using Intellij's markdown preview the tags are used to show preformatting.

![image](https://user-images.githubusercontent.com/12058921/122157291-f4673280-cead-11eb-99f5-a5f0154a16c0.png)